miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultLibraryDilutionService.java

```diff
@@ -57,7 +57,19 @@ public LibraryDilution get(long dilutionId) throws IOException {
     return dilution;
   }
 
+  private boolean isTargetedSequencingInLibrary(LibraryDilution dilution) {
+    return dilution.getLibrary().getKitDescriptor().getTargetedSequencing().contains(dilution.getTargetedSequencing());
+  }
+
+  private void validateTargetedSequencing(LibraryDilution dilution) {
+    if (!isTargetedSequencingInLibrary(dilution)) {
```

Comment:
Targeted sequencing may also be null if
* library is not detailed (Use `LimsUtils#isDetailedLibrary`)
* library's LibraryDesignCode does not require targetedSequencing (Use LibraryDesignCode#isTargetedSequencingRequired)

Reply:
@djcooke Thanks for the clarification (this may also explain why a plain sample test is failing in the integration tests).

In either case above, targeted sequencing *should* be null, correct?

Reply:
For plain library, it should be null. If the design code allows null, it doesn't *have* to be null though.

Reply:
Since targeted sequencing and kit descriptor are available on plain dilutions and libraries, I don't think it's necessarily the case that it will be null for plain libraries. Plain libraries don't have required kit descriptors though, whereas detailed libraries do. Or is there something I'm missing @djcooke ?

Reply:
The UI doesn't currently allow a user enter targeted sequencing for plain libraries. That could change though, and it doesn't really hurt anything if a plain dilution has targeted sequencing set.

Could change this instead to
```
if tarseq is null
  make sure null is ok
else
  make sure tarseq is compatible with kit
```